features/security/src/main/feature/feature.xml

```diff
@@ -21,694 +21,317 @@
-        <feature>security-filter-login</feature>
-        <feature>security-filter-csrf</feature>
-        <feature>security-handler-basic</feature>
-        <feature>security-oidc-realm</feature>
-        <feature>security-handler-oidc</feature>
-        <feature>security-handler-oauth</feature>
-        <feature>security-handler-saml</feature>
-        <feature>security-filter-authz</feature>
-        <feature>security-pdp-authz</feature>
-        <feature>security-servlet-whoami</feature>
-        <feature>security-servlet-session-expiry</feature>
-        <feature>security-servlet-web-socket</feature>
-        <feature>security-realm-saml</feature>
-        <feature>security-guest-realm</feature>
-        <feature>security-claims-property</feature>
+    <feature name="oidc-auth" version="${project.version}"
+             description="OIDC features and bundles.">
         <feature>security-core-services</feature>
-    </feature>
 
-    <feature name="secure-boot" version="${project.version}"
-             description="Verifies that the installation is located in a secure directory.">
-        <bundle>mvn:ddf.platform.security/secure-boot/${project.version}</bundle>
-    </feature>
-
-    <feature name="security-core" version="${project.version}"
-             description="The Security Application provides Authentication, Authorization, and Auditing services.\nThey comprise both a framework that developers and integrators can extend and also a reference implementation that can be used which meets security requirements.::Security">
-        <feature>web-container</feature>
-        <feature>security-core-impl</feature>
-        <feature>security-policy-context</feature>
-        <feature>security-web-sso-defaults</feature>
-        <feature>security-guest</feature>
-        <feature>security-command-listener</feature>
+        <bundle>mvn:ddf.security.realm/security-realm-oidc/${project.version}</bundle>
+        <bundle>mvn:ddf.security.handler/security-handler-oidc/${project.version}</bundle>
+        <bundle>mvn:ddf.security.handler/security-handler-oauth/${project.version}</bundle>
     </feature>
 
-    <!-- Start of app features -->
-
-    <repository>mvn:ddf.features/camel/${project.version}/xml/features</repository>
+    <feature name="saml-auth" version="${project.version}"
+             description="SAML features and bundles.">
+        <feature>security-core-services</feature>
 
-    <feature name="platform-http-proxy" version="${project.version}"
-             description="HTTP to HTTPS proxy.">
-        <feature>camel-jetty9</feature>
-        <feature>security-core-api</feature>
-        <feature>camel-blueprint</feature>
-        <feature>camel-http</feature>
-        <bundle>mvn:org.codice.thirdparty/commons-httpclient/3.1.0_1</bundle>
-        <bundle>mvn:ddf.platform/platform-http-proxy/${project.version}</bundle>
-    </feature>
+        <!--  TODO (https://github.com/codice/ddf/issues/5327)
```

Comment:
❓ This issue is shown as closed in github. Should it be re-opened?

Reply:
It's still an issue, but I'm not sure when we are planning to get around to resolving it. I'm not sure what the best practice is in this case.